lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -173,14 +175,15 @@ type ListAddresses t = "wallets"
   See also: https://input-output-hk.github.io/cardano-wallet/api/#tag/Wallets
 -------------------------------------------------------------------------------}
 
-type Wallets =
+type Wallets n =
     DeleteWallet
     :<|> GetWallet
     :<|> ListWallets
     :<|> PostWallet
     :<|> PutWallet
     :<|> PutWalletPassphrase
     :<|> GetUTxOsStatistics
+    :<|> SelectCoins n
```

Comment:
**[change]**

Move `SelectCoins` API into a new group `CoinSelections` as described in the original U/S

**Why?**

Just a bit of organization to avoid surcharging the wallets endpoints. Mostly everything relates to a wallet, but some endpoints are better placed under some other categories (like transactions for instance).

Reply:
Done! See https://github.com/input-output-hk/cardano-wallet/pull/1152/commits/f8abada2d1cb8da1e5adcc071cbdd24c4e0594ce.